### PR TITLE
Add link claiming

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

Add link to the transaction of the claiming

image

  # To Test

1. Claim for any account
2. Make sure the link displayed takes you to Etherscan

### CODE REVIEW

File: src/custom/pages/Claim/ClaimingStatus.tsx [hunk 1]
```diff
@@ -6,12 +6,21 @@ import { useClaimState } from 'state/claim/hooks'
 import { useActiveWeb3React } from 'hooks/web3'
 import CowProtocolLogo from 'components/CowProtocolLogo'
 import Circle from 'assets/images/blue-loader.svg'
+import { ExplorerDataType, getExplorerLink } from 'utils/getExplorerLink'
+import { useAllClaimingTransactions } from '@src/custom/state/enhancedTransactions/hooks'
```
Comment (alfetopito) [inline src/custom/pages/Claim/ClaimingStatus.tsx:10]: Path!

```suggestion
import { useAllClaimingTransactions } from 'state/enhancedTransactions/hooks'
```

**Review by alfetopito — APPROVED** [review]

**Comment by elena-zh** [comment]
Hey @anxolin , could you please help me to get a preview link?

**Comment by anxolin** [comment]
Merging, let me know if anyone finds issues with this one

**Comment by elena-zh** [comment]
The link navigates to the transaction hash in Rinkeby, but I'd change its description to 'View transaction on Etherscan', and 'View transaction on Blockscount' (for xDAI), as 'Explorer' links in CowSwap navigate to the Cow Protocol explorer.

**Comment by anxolin** [comment]
> The link navigates to the transaction hash in Rinkeby, but I'd change its description to 'View transaction on Etherscan', and 'View transaction on Blockscount' (for xDAI), as 'Explorer' links in CowSwap navigate to the Cow Protocol explorer.

Good point! although i guess is nitpic, cause we used explorer as something that would fit all. Etherscan, Blockscout, and CoW Explorer are all explorers. However, taking note for improvements.

Thanks!

**Pull request merged**